nixos/modules/services/mail/mailman.nix

```diff
@@ -241,8 +246,8 @@ in {
         mailmanCfg=$mailmanDir/mailman-hyperkitty.cfg
         mailmanWebCfg=$mailmanWebDir/settings_local.json
 
-        install -m 0700 -o mailman -g nogroup -d $mailmanDir
-        install -m 0700 -o ${cfg.webUser} -g nogroup -d $mailmanWebDir
+        install -m 0700 -o mailman -g mailman -d $mailmanDir
+        install -m 0700 -o ${cfg.webUser} -g mailman -d $mailmanWebDir
```

Comment:
I don’t understand the purpose of this change. The directories are not group-readable.

Reply:
Oof sorry, I guess I forgot to include that part. Fixing.

Reply:
Also for what it's worth this isn't the _only_ way to do this.  Another workaround would be to set mod 0711 on the directories, and make some subparts of them world-readable, but using membership in a shared group seems slightly cleaner to me.